### PR TITLE
Added a new test case for the issue #7724 in polys

### LINKED ISSUE #7724: roots should find the roots of x**4*I + x**2 + I
```
In [34]: roots(x**4*I + x**2 + I, x)
Out[34]: {}
```

surprisingly if we replace `I` with variable coefficients we get an answer.

```
In [35]: roots(x**4*a + x**2 + b, x)
Out[35]: 
{sqrt(2)*sqrt(-sqrt(-4*a*b + 1)/a - 1/a)/2: 1,
 sqrt(2)*sqrt(sqrt(-4*a*b + 1)/a - 1/a)/2: 1,
 -sqrt(2)*sqrt(sqrt(-4*a*b + 1)/a - 1/a)/2: 1,
 -sqrt(2)*sqrt(-sqrt(-4*a*b + 1)/a - 1/a)/2: 1}
```

### PR DESCRIPTION
**Pull request by Jonty16117**
#### Added a new test case for the issue #7724 in polys

Fixes #7724

#### Brief description of what is fixed or changed
For the issue #7724 , I added a new test case.

NO ENTRY

### CODE REVIEW

File: sympy/polys/tests/test_polyroots.py [hunk 1]
```diff
@@ -73,6 +73,13 @@ def test_roots_quadratic():
         roots = roots_quadratic(f)
         assert roots == _nsort(roots)
 
+
+def test_issue_7724():
+    eq = Poly(x**4*I + x**2 + I, x)
+    roots_d = roots(eq)
+    assert len(roots_d) == 4
```
Comment (oscarbenjamin) [inline sympy/polys/tests/test_polyroots.py]: I think it would be better to assert what the roots are then just how many there are

**Comment by smichr** [comment]
NO ENTRY is ok for this sort of coverage addition. There is nothing that will change for the user.

File: sympy/polys/tests/test_polyroots.py (hunk 1 above)
Comment (Jonty16117) [reply sympy/polys/tests/test_polyroots.py]: Ok... I'll commit it asap

**Comment by Jonty16117** [comment]
> NO ENTRY is ok for this sort of coverage addition. There is nothing that will change for the user.

Tx...😊

**Pull request merged**